python/modem-firmware-1.3+/README.md

```diff
@@ -380,20 +382,20 @@ The parsed data is displayed in the output.  Providing the COSE string to this s
 
 
 ```
-usage: modem_credentials_parser.py [-h] [-k KEYGEN] [-a ATTEST]
+usage: modem_credentials_parser.py [-h] [-k KEYGEN] [-a ATTEST] [-s] [-p PATH] [-f FILEPREFIX]
 
-Modem Credentials Parser
+Modem Credential Parser
```

Comment:
Are you also renaming the script to `modem_credential_parser.py`?

Reply:
no, just matching the -h output with the readme

Reply:
updated to match filename 😝